packages/pose-core/src/factories/setter.ts

```diff
@@ -227,8 +217,16 @@ const createPoseSetter = <V, A, C, P>(
           setValue,
           setValueNative
         );
+        baseTransitionProps.elementStyler.render();
```

Comment:
this is needed because otherwise those 2:
https://github.com/Popmotion/popmotion/blob/916df7350307d3127bb2dbdd9ea721e03888791c/packages/popmotion-pose/src/dom/unit-conversion.ts#L55
https://github.com/Popmotion/popmotion/blob/916df7350307d3127bb2dbdd9ea721e03888791c/packages/popmotion-pose/src/dom/unit-conversion.ts#L81
would return outdated computed styles.

Most likely the same happens in flip.ts

Reply:
Hm, seems like elementStyler is DOM-specific - so this call has to be moved out of `pose-core`.

How would you suggest doing this?

Reply:
Ugh. Yeah this makes sense. 

FLIP was a mistake.

You can add "slots" to `pose-core`, that pose flavours can hook into. The same way there's

```
if (transformPose) nextPose = transformPose(nextPose, next, state);
```

You could add an optional function to the `pose-core` factory that's something like "forceRender", kinda generic, that in `popmotion-pose` is configured to be `props.elementStyler.render()` and you call it if you've set `applyAtStart`

Reply:
Cool, was thinking about such hook myself, couldnt settle on a name - `forceRender` sounds nice 👌 

> FLIP was a mistake.

Why? :p do you mean using this technique or its current implementation?

Reply:
Also - the quirk of this solution is that `applyAtStart` cant be transformed (while ie `applyAtEnd` actually is transformed in some cases). It is how it is though, cant figure out any legit use case for transforming `applyAtStart` anyway.

Reply:
Mostly because I think 80% of all the bugs we've seen in Pose have been around FLIP!